### DISCUSSION TITLE
Webots cloud server issue

### DISCUSSION THREAD

**Discussion by jmemon16** [cyberbotics/webots#6724] [General]
New user here. I cannot run any simulation on [webots.cloud](http://webots.cloud/) and don't see any server listed [webots.cloud - server](https://webots.cloud/server). I get following prompt when I try to run a simulation online. Is this normal?  Are there any test servers available to test out functionality? Am I supposed to create my own server?
"Session server error: No simulation server available at the moment"

**Comment by omichel** [comment]
Yes, this functionality is currently down. You are free to create your own server and use it to run your simulations online though.
